package.json

```diff
@@ -244,7 +244,7 @@
 	"scripts": {
 		"build": "npm run build:packages && wp-scripts build",
 		"build:analyze-bundles": "npm run build -- --webpack-bundle-analyzer",
-		"build:package-types": "node ./bin/packages/validate-typescript-version.js && tsc --build",
+		"build:package-types": "node ./bin/packages/validate-typescript-version.js",
```

Comment:
This needs to be reverted before merging and all issues with typescript are solved